app/controllers/discourse_preset_topic_composer/preset_tag_groups_controller.rb

```diff
@@ -7,7 +7,14 @@ class PresetTagGroupsController < ::ApplicationController
     def search_tags_by_tag_group
       tag_group = params[:tag_group]
       tag_group = CGI.unescape(tag_group)
-      tags = TagGroup.visible(guardian).find_by(name: tag_group)&.tags || []
+
+      if SiteSetting.tags_sort_alphabetically
+        tag_order = "name ASC"
+      else
+        tag_order = "public_topic_count DESC"
+      end
```

Comment:
```suggestion
      tag_order =
        if SiteSetting.tags_sort_alphabetically
          "name ASC"
        else
          "public_topic_count DESC"
        end
```
feels better, what do you think?